meta.yaml

```diff
@@ -0,0 +1,134 @@
+{% set name = "flytekit" %}
+{% set version = "0.24.0b0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 21c1560de9a3af594ac542f31d6888f5232540f5338f277493c31d4e49c713a1
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - pyflyte-execute=flytekit.bin.entrypoint:execute_task_cmd
+    - pyflyte-fast-execute=flytekit.bin.entrypoint:fast_execute_task_cmd
+    - pyflyte-map-execute=flytekit.bin.entrypoint:map_execute_task_cmd
+    - pyflyte=flytekit.clis.sdk_in_container.pyflyte:main
+    - flyte-cli=flytekit.clis.flyte_cli.main:_flyte_cli
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - checksumdir >=1.2.0
```

Comment:
is there no way to re-use the requirements? I prefer not to duplicate it

Reply:
I am afraid that the requirements.txt can not be reused here. This `meta.yaml` is the only file needed when submitting to https://github.com/conda-forge/staged-recipes.
However, I am not sure whether it can be simplified or not. I am looking at [examples](https://github.com/conda-forge/staged-recipes/pulls?q=is%3Apr+is%3Aclosed) here, which are closed PRs at conda-forge.
For instance, this one https://github.com/conda-forge/staged-recipes/pull/16139/files, duplicate its own requirements.txt in https://github.com/coiled/dask-snowflake